src/components/base/SentryErrorBoundary.tsx

```
@@ -0,0 +1,97 @@
+}
+
+// Create a higher-order component version of the error boundary
+export function withSentryErrorBoundary<P extends object>(
+  Component: React.ComponentType<P>,
+  fallback?: Props['fallback'],
+): React.FC<P> {
+  return (props: P) => (
+    <SentryErrorBoundary fallback={fallback}>
+      <Component {...props} />
+    </SentryErrorBoundary>
+  )
```

ESLint

[error] Component definition is missing display name

(react/display-name)